### ISSUE TITLE
actuators of Somfy is always closed

### ISSUE THREAD

**Issue by Beat540** [dubocr/homebridge-tahoma#139]
Hello Romain, thanks for the great plugin Tahoma.
Unfortunately, the status of three actuators of Somfy is always closed even if the blinds are open. The behavior does not change with the command "stateless" true or "cyclic" true. 

Thanks for your feedback.

**Comment by dubocr** [comment]
Hi,

Please log there : https://dev.duboc.pro/tools/setup

**Comment by Beat540** [comment]
Hi Romain
the blinds "1 Dach", "2 Ost" and "3 Ost" have this Problem. 
Thanks for your support

**Comment by dubocr** [comment]
Hi,
May they are just reversed as there is a state "core:TargetClosureState". On is et 104%, it's strange but you can try this config :
```
"PositionableHorizontalAwningUno" : { "reverse": true },
"PositionableScreenUno" : { "reverse": true }
```

**Comment by Beat540** [comment]
Hi Romain
Ich habe folgende Zeilen eingefügt in den Drei Storen, die das Problem haben:

      "1 Dach" : {
        "PositionableScreenUno" : {"reverse" : true},
        "PositionableHorizontalAwningUno" : {"reverse" : true}
      },
      "3 Ost" : "PositionableScreenUno" : {"reverse" : true},
        "PositionableHorizontalAwningUno" : {"reverse" : true}
      },
      "2 Ost" : {"PositionableScreenUno" : {"reverse" : true},
        "PositionableHorizontalAwningUno" : {"reverse" : true}
      }

Die Storen bleiben nach dem Neustart und dem Befehl «Alle auf» auf dem Status «Geöffnet». Nach ca. 10 Min. (RefreshPeriod?) zeigt die Homebridge die 3 Storen wieder als «geschlossen» an – sie sind aber offen!

Muss ich allenfalls die zwei Befehle für alle Storen eingeben? Dann würden aber ev. die Storen 4/5/6 nicht mehr funktionieren, die jetzt richtig funktionieren.

Sorry wenn ich in Deutsch schreibe, hoffe Du versteht es?

Danke und Gruss
Beat Ineichen


Von: Romain DUBOC <notifications@github.com>
Gesendet: Mittwoch, 11. Dezember 2019 20:13
An: dubocr/homebridge-tahoma <homebridge-tahoma@noreply.github.com>
Cc: Beat Ineichen <beat@ineichen.online>; Author <author@noreply.github.com>
Betreff: Re: [dubocr/homebridge-tahoma] actuators of Somfy is always closed (#139)


Hi,
May they are just reversed as there is a state "core:TargetClosureState". On is et 104%, it's strange but you can try this config :

"PositionableHorizontalAwningUno" : { "reverse": true },

"PositionableScreenUno" : { "reverse": true }

**Comment by dubocr** [comment]
Hi Beat,

No I don't understand a word :-). Thanks Google translate.
Don't put the config inside your device name, but at the same level than plugin config.

```
{
   "Platform": "Tahoma",
   "user": "...",
   ...
   "PositionableHorizontalAwningUno" : { "reverse": true },
   "PositionableScreenUno" : { "reverse": true }
}
```

**Comment by Beat540** [comment]
Hi Romain
the status is now maintained with the commands, but the status is incorrect for the affected blinds. Open = closed?!

regards beat

Von: Romain DUBOC <notifications@github.com>
Gesendet: Samstag, 14. Dezember 2019 11:30
An: dubocr/homebridge-tahoma <homebridge-tahoma@noreply.github.com>
Cc: Beat Ineichen <beat@ineichen.online>; Author <author@noreply.github.com>
Betreff: Re: [dubocr/homebridge-tahoma] actuators of Somfy is always closed (#139)


Hi Beat,

No I don't understand a word :-). Thanks Google translate.
Don't put the config inside your device name, but at the same level than plugin config.

{

   "Platform": "Tahoma",

   "user": "...",

   ...

   "PositionableHorizontalAwningUno" : { "reverse": true },

   "PositionableScreenUno" : { "reverse": true }

}

**Comment by dubocr** [comment]
Could you close your blinds then log to https://dev.duboc.pro/tools/setup again to check the value of core:TargetClosureState.

Thanks

**Comment by Beat540** [comment]
Hi Romain
Ok, 1 Dach, 2 Ost and 3 Ost are close


Von: Romain DUBOC <notifications@github.com>
Gesendet: Sonntag, 15. Dezember 2019 13:29
An: dubocr/homebridge-tahoma <homebridge-tahoma@noreply.github.com>
Cc: Beat Ineichen <beat@ineichen.online>; Author <author@noreply.github.com>
Betreff: Re: [dubocr/homebridge-tahoma] actuators of Somfy is always closed (#139)


Could you close your blinds then log to https://dev.duboc.pro/tools/setup again to check the value of core:TargetClosureState.

Thanks

**Comment by dubocr** [comment]
Is the state good at init ? Command and state are both OK. Can you send me homebridge logs and associated actions and what's wrong.

Thks

**Comment by Beat540** [comment]
Hi Romain
1 Dach:
the status is wrong, open = close, after update from version 0.3.20 to version 0.3.21
                I have inserted the following command:
    {
      "1 Dach" : {
        "reverse" : true
      },
sometimes core:TargetClosureState 104! Sometimes core:TargetClosureState 100 (blinds are open)

2 Ost und 3 Ost:
the status (open) is in Homebridge correct first and core:TargetClosureState 0
after ~10-20 min. Status in Homebridge falls back to closed and core:TargetClosureState 100! (blinds are open)

Thanks for your support



Von: Romain DUBOC <notifications@github.com>
Gesendet: Sonntag, 15. Dezember 2019 19:26
An: dubocr/homebridge-tahoma <homebridge-tahoma@noreply.github.com>
Cc: Beat Ineichen <beat@ineichen.online>; Author <author@noreply.github.com>
Betreff: Re: [dubocr/homebridge-tahoma] actuators of Somfy is always closed (#139)


Is the state good at init ? Command and state are both OK. Can you send me homebridge logs and associated actions and what's wrong.

Thks

**Comment by dubocr** [comment]
State polling improved in latests releases
